pkg/plugins/runtime/k8s/controllers/pod_controller.go

```diff
@@ -233,6 +245,9 @@ func (r *PodReconciler) SetupWithManager(mgr kube_ctrl.Manager) error {
 		Watches(&kube_source.Kind{Type: &mesh_k8s.Dataplane{}}, &kube_handler.EnqueueRequestsFromMapFunc{
 			ToRequests: &DataplaneToSameMeshDataplanesMapper{Client: mgr.GetClient(), Log: r.Log.WithName("dataplane-to-dataplanes-mapper")},
 		}).
+		Watches(&kube_source.Kind{Type: &kube_core.ConfigMap{}}, &kube_handler.EnqueueRequestsFromMapFunc{
+			ToRequests: &ConfigMapToPodsMapper{Client: mgr.GetClient(), Log: r.Log.WithName("configmap-to-pods-mapper")},
+		}).
```

Comment:
are we watching all config maps in kubernetes cluster and fire reconciliation process on change? Can we at least to this on config map with DNS only?

Reply:
Fixed, now fire reconciliation process only on changing of `kuma-dns-vips` configmap